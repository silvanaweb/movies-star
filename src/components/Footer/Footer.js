import React from "react";

const Footer = () => <footer style={{ height: "3rem" }} />;

export { Footer };
