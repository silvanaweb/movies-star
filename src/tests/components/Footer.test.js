import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import { Footer } from "../../components/Footer/Footer";

test("should render Footer correctly", () => {
  const renderer = new ShallowRenderer();
  const result = renderer.render(<Footer />);
  expect(result).toMatchSnapshot();
});
