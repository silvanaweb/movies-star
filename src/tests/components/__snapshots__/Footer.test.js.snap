// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer correctly 1`] = `
<footer
  style={
    Object {
      "height": "3rem",
    }
  }
/>
`;
